### DISCUSSION TITLE
change itwin dark theme from grey to green

### DISCUSSION THREAD

**Discussion by Rahul-0108** [iTwin/iTwinUI#1183] [Q&A]
We want to change the dark theme from grey to green, how this can be achieved ?? 
our requirement is to use green theme for dark theme , because our other apps are in green theme, so we should have similar theme in all our apps
Any sample code or anything will be helpful

**Comment by Rahul-0108** [comment]
@mayank99 , any idea ??

**Comment by mayank99** [comment]
Do you have screenshots of what exact parts you would like to change? Also what version of iTwinUI are you using?

Once you determine those things, the best path forward will be to override our color CSS variables.

**Reply by Rahul-0108** [reply]
I saw in the node modules file
![image](https://user-images.githubusercontent.com/53996840/229806910-684ab7fe-a823-47f6-b71a-ac682c309332.png)

so overriding these variable in our app code will make it work right ??
We want to migrate all the components from grey to green in itwinui
![image](https://user-images.githubusercontent.com/53996840/229807819-74401e10-7bb0-48b6-97d7-16b5cbbb3dc3.png)
our versions
![Uploading image.png…]()

**Reply by Rahul-0108** [reply]
![image](https://user-images.githubusercontent.com/53996840/229808639-ed3a4b18-3b18-4f94-a456-544349eb0b8d.png)

**Reply by mayank99** [reply]
Those are Sass variables; updating them will have no effect because they are already consumed/compiled away.

What you want is to update these CSS variables: https://unpkg.com/browse/@itwin/itwinui-css@0.63.1/css/global.css

if you only want to change dark theme, then you can do it like this:
```scss
html.iui-theme-dark {
  --iui-color-background-1: green;
  --iui-color-background-2: hotpink;
  // ...
}
```

---

side note: I really hope you have a good reason to make these updates, and you have consulted with the UX team. Making everything green is generally a bad idea.
